content/bd/0004-relational-model-conversion.md

```diff
@@ -406,8 +395,8 @@ da entidade de multiplicidade 1 e obrigatória.
 
 ### Generalizações/Especializações
 
-Pegando no exemplo anterior de _pessoa_, _professor_ e _aluno_, como podemos
-converter este modelo para o modelo relacional?
+Pegando no exemplo de _Pessoa_, _Professor_ e _Aluno_, como podemos
+converter este modelo E-A para o modelo relacional?
```

Comment:
```suggestion
converter este diagrama de modelo E-A para o modelo relacional?
```
The E-A Model is the spec, this is just a diagram.
Confirmed this is the language used in the project's Part 1 statement.